packages/@aws-cdk/aws-s3/README.md

```diff
@@ -480,15 +480,28 @@ by deploying with CDK version `1.126.0` or later **before** switching this value
 
 ```ts
 const bucket = new s3.Bucket(this, 'MyBucket', {
-  transferAcceleration: true,
+   transferAcceleration: true,
 });
 ```
 
 To access the bucket that is enabled for Transfer Acceleration, you must use a special endpoint. The URL can be generated using method `transferAccelerationUrlForObject`:
 
 ```ts
 const bucket = new s3.Bucket(this, 'MyBucket', {
-  transferAcceleration: true,
+   transferAcceleration: true,
 });
 bucket.transferAccelerationUrlForObject('objectname');
 ```
+
+## Intelligent Tiering
+
+[Intelligent Tiering](https://docs.aws.amazon.com/AmazonS3/latest/userguide/transfer-acceleration.html) can be configured to enable fast, easy, and secure transfers of files over long distances:
+
+```ts
+    new s3.Bucket(this, 'MyBucket', {
+   intelligentTieringConfigurations: [{
+      name: 'foo',
```

Comment:
I think you could include other props here to make the usefulness of this feature more evident, like `prefix`, `archiveAccessTierTime`.

Reply:
updated